src/svg2pdf.js

```diff
@@ -231,6 +231,30 @@ SOFTWARE.
     return p2;
   };
 
+  /**
+   * 
+   * @param {object} values 
+   * @constructor
+   * @property {AttributeState} attributeState  Keeps track of parent attributes that are inherited automatically
+   * @property {ReferencesHandler} refsHandler  The handler that will render references on demand
+   * @property {jspdf.Matrix} transform The current transformation matrix
+   * @property {boolean} withinClipPath 
+   * @property {boolean} withinDefs True if we are top-level within a defs node, so the target can be switched to an pdf form object
+   */
+  function Context(values) {
+    values = values || {};
+    this.attributeState = values["attributeState"] ? values["attributeState"].clone() : AttributeState.default();
+    this.transform = values["transform"] || _pdf.unitMatrix;
+    this.refsHandler = values["refsHandler"] || null;
+    this.withinClipPath = values["withinClipPath"] || false;
+    this.withinDefs = values["withinDefs"] || false;
+  }
+
+  Context.prototype.set = function (property, value) {
```

Comment:
I don't think we need this method. Why not simply write this instead:
```
var childContext = new Context(context);
childContext.foo = foo
```

Reply:
There are a few reasons (but yes, I don't know if we really can count them as such... ):
 \+ we _save_ **one** line of code (each time)
 \+ we don't have to search for new names
 \+(or rather '-') we complicate the code (actually this one is not a reason, it's more a consequence)
Okay, okay, I'll change it...